packages/expo-linking/android/src/main/java/expo/modules/linking/ExpoLinkingModule.kt

```diff
@@ -0,0 +1,38 @@
+package expo.modules.linking
+
+import android.net.Uri
+import androidx.core.os.bundleOf
+import expo.modules.kotlin.modules.Module
+import expo.modules.kotlin.modules.ModuleDefinition
+
+class ExpoLinkingModule : Module() {
+  companion object {
+    var initialURL: Uri? = null
+    var onURLReceived: ((Uri?) -> Unit)? = null
```

Comment:
Unfortunately, I'm not sure if handling only one listener is enough. What if the dev menu also tries registering its receiver, overriding the one from the user app? It would be nice to support multiple observers, even from different contexts.

Reply:
Ok, I'm now keeping a set of callbacks to support this.